crates/config/Cargo.toml

```diff
@@ -29,7 +29,9 @@ Inflector = "0.11"
 number_prefix = "0.4"
 once_cell = "1"
 regex = "1"
-reqwest = { version = "0.11", default-features = false }
+reqwest = { version = "0.12", default-features = false, features = [
+    "default-tls",
```

Comment:
we don't want this,
why is this enabled?

Reply:
this should be there with any tls feature, 
ref:

https://github.com/foundry-rs/foundry/blob/a16714ed40f733013d7a80f4f969564175c3318e/crates/forge/Cargo.toml#L107-L113